docs/api/environment-variables.md

```diff
@@ -120,6 +120,15 @@ debugging purposes.
 
 Prints Chrome's internal logging to the console.
 
+### `ELECTRON_DEBUG_DRAG_REGIONS`
+
+Adds coloration to draggable regions on `BrowserViews` on macOS - draggable regions will be colored
+green and non-draggable regions will be colored green to aid debugging.
+
+### `ELECTRON_DEBUG_NOTIFICATIONS`
+
+Adds extra debugging logs to Notification lifecycles on macOS to aid in debugging.
```

Comment:
Question: is this specifically for the main process `Notification` module? Should we add backticks and link the docs?
```suggestion
Adds extra debugging logs to [`Notification`](./notification.md) lifecycles on macOS to aid in debugging.
```

Reply:
It would be useful to note what kinds of logs this can produce and what to expect from this variable!